definitions/xml2js.d.ts

```diff
@@ -0,0 +1,99 @@
+// Type definitions for node-xml2js
+// Project: https://github.com/Leonidas-from-XIV/node-xml2js
+// Definitions by: Michel Salib <https://github.com/michelsalib>, Jason McNeil <https://github.com/jasonrm>, Christopher Currens <https://github.com/ccurrens>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```

Comment:
No yet, as it is unclear whether we'll continue using this library or switch to another (xmlreader).